### DISCUSSION TITLE
Datacenter view feature requiest

### DISCUSSION THREAD

**Discussion by dimasmir** [Molkobain/itop-datacenter-view#15] [Ideas]
Hi.

I would like to offer a feature request. It would be cool to be able to see all racks scheme in the same location.
I do not have enough of this feature in my work.

Thanks and have a nice day.

**Comment by Molkobain** [comment]
Hello @dimasmir , 

Thanks for the feedback! Actually this feature is already on the roadmap but it will only be part of the **extended version**.

The idea is to have in a Location a new tab with the top view of the room "grid" with all the racks but also be able to represent A/Cs and such.
In a Rack there will also be a new tab, showing the same top view of the room but to highlight where the rack is located. It might also contain a front view of the rack aisle to highlight it there as well. Same thing with a DatacenterDevice, to highlight it in the rack / aisle.

Feel free to let me know what you think about this.

Cheers,
Guillaume

**Comment by dimasmir** [comment]
Sounds very good. Good for me, that I've already Extended version user.

**Comment by dimasmir** [comment]
Good day.

Do you have any news about this feature. 
My support contract is expiring and it is interesting to renew when feature will be released.

**Comment by Molkobain** [comment]
Hello Dimasmir,

Unfortunately this feature isn't going to be release by the end of 2022. For now we are having a few setbacks with the "full depth enclosures" feature and are focusing on it before releasing anything else. But I'll keep you udpated when the location stuff is done. :)

Guillaume

**Comment by lazarobaca** [comment]
Devices in inverse ordes, depending of the branch, there are Racks that are numbered from up to down, can It be in come chek control for make the view according with the physical numbers in the rack? thanks
